src/proof/alethe/alethe_post_processor.h

```diff
@@ -99,6 +99,50 @@ class AletheProofPostprocessCallback : public ProofNodeUpdaterCallback
                            CDProof& cdp);
 };
 
+/**
+ * Final callback class used by the Alethe converter to add the last step to a
+ * proof in the following two cases. The last step should always be printed as
+ * (cl).
+ *
+ * 1. If the last step of a proof which is false is reached it is printed as (cl
+ *    false).
+ * 2. If one of the assumptions is false it is printed as false.
+ *
+ * Thus, an additional resolution step with (cl (not true)) has to be added to
+ * transfer (cl false) into (cl).
```

Comment:
Instead of adding such a step couldn't you just change the third argument, the term to be printed, so that you had `(cl)` as the printed conclusion? If I remember correctly this is in line with the semantics of the resolution rule?

Reply:
I will consider this in a separate PR after the alethe branch is fully merged since there were some problems with special cases when we tried to do this differently previously.